src/main/webapp/app/components/icons/OncoKBEdit.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { IconBaseProps } from 'react-icons';
+import { FaEdit } from 'react-icons/fa';
+import { IconContext } from 'react-icons/lib';
+
+export default function OncoKBEdit(props: IconBaseProps) {
```

Comment:
This should be moved to shared/icons, and should be named as EditIcon. Same goes to CommentIcon, it should be moved as well.